### PR TITLE
Update README with specs, license, build step & prereqs

### LINKED ISSUE #997: Expand node setup documentation
#### Summary
We want to allow interested people and third-parties to either setup their own cluster or connect to our testnet. 

#### Possible solution
Expand the existing `README` with prerequisites, system requirements and an explanation for some of the main configuration options.

We can provide a step-by-step guide, similar to what was used during ITN1, and post it on the wiki. Or we could provide it as a markdown file in the repository. It is important that the guide contains details on how to check that the node is working as expected.

### PR DESCRIPTION
**Pull request by HDauven**
Resolves #997

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,53 @@
...
+| :--- | :--- | :--- | :--- |
+| 4 cores; 2 GHz | 4 GB | 250 GB | 10 Mbps |
+
 ## Build and Tests
 
-To run tests:
+To build `rusk` from source, Rust, GCC and Clang are required. Once the dependencies are installed, you can simply run the following command to compile everything:
 
+```bash
+make
```
Comment (herr-seppia) [inline README.md:36]: .env should be loaded before `make`
That because make keys requires it, and the keys are required to build circuits...

**Review by herr-seppia — DISMISSED** [review]
LGTM

Just double check if `source .env` is required

File: README.md (hunk 1 above)
Comment (HDauven) [reply README.md:37]: Added it to all sections that indirectly call `make keys`, for consistency. Tnx!

**Review by ureeves — APPROVED** [review]
LGTM

**Pull request merged**